src/model/operation/convolution.cc

```diff
@@ -218,7 +221,7 @@ CudnnConvHandle::CudnnConvHandle(const Tensor &input,
 #endif
                                              ));
   if (CUDNN_MAJOR >= 7 && groups > 1) {
-    CUDNN_CHECK(cudnnSetConvolutionGroupCount(conv_desc, groups));
+    //CUDNN_CHECK(cudnnSetConvolutionGroupCount(conv_desc, groups));
```

Comment:
why comment it out?

Reply:
I used the wrong cudnn version before. I have uncommented it